src/highlevel/forward_mode.jl

```diff
@@ -6,41 +6,53 @@ struct ForwardDiffJacobianCache{CO, CA, J, FX, X} <: AbstractMaybeSparseJacobian
     x::X
 end
 
+__getfield(c::ForwardDiffJacobianCache, ::Val{:jac_prototype}) = c.jac_prototype
+
 struct SparseDiffToolsTag end
 
+function ForwardDiff.checktag(::Type{<:ForwardDiff.Tag{<:SparseDiffToolsTag, <:T}}, f::F,
+        x::AbstractArray{T}) where {T, F}
+    return true
+end
```

Comment:
```suggestion
```

This isn't necessary if done right

Reply:
https://github.com/JuliaDiff/ForwardDiff.jl/blob/d3002093beb88ff0b98ed178377961dfd55c1247/src/config.jl#L42-L43 would trigger only if we don't wrap it into a Tag object